dp-game-webapp/app/game/play/questions.tsx

```diff
@@ -321,6 +348,16 @@ function AnswerButtons({
       >
         Decrease <ArrowDownCircleIcon className="h-4 lg:h-8 w-auto ml-2" />
       </button>
+
+      <button
+        className={`h-8 lg:h-12 lg:py-2 px-1 lg:px-4 text-base text-sm lg:text-lg font-medium text-white hover:bg-teal-700 rounded-lg flex items-center justify-between ${
+          answer === Answer.MaintainSpend ? "bg-teal-700" : "bg-teal-400"
+        }`}
+        onClick={() => handleMaintainSpend(questionIndex)}
+      >
+        Maintain <MinusCircleIcon className="h-4 lg:h-8 w-autho ml-2" />
```

Comment:
I thought equal would be a better visualization of maintain vs minus

Reply:
I'm not particularly opinionated here. I was thinking this looks "flat". Equal would make sense if we were using +/- for increase/decrease, but it could also work with the arrows.

Reply:
well, even better answer. there isn't a [heroicon](https://heroicons.com/) with a equal sign in a circle ¯\_(ツ)_/¯